app/views/pets/show.html.erb

```diff
@@ -2,7 +2,13 @@
   <%= image_tag ( @pet.image ) %>
   <h2><%= @pet.name %></h2>
   Status: <%= @pet.adopt_status %><br/>
-  <%= link_to "Favorite Pet", "/favorite/#{@pet.id}", method: :patch %>
+
+  <% if favorite.check_pet("#{@pet.id}") %>
```

Comment:
This is cool that we can do this type of logic here.  Not 100% sure this is where it belongs, but I'm not sure where it does if not here.